src/diffpy/utils/parsers/loaddata.py

```diff
@@ -16,29 +16,47 @@
 import numpy
```

Comment:
Same as loaddata_headers

Reply:
I am not sure what this means, but I think it means you are just moving code over from the old function?  Anyway, no need to respond unless you need a response from me and I am not providing it.